aggregating-Apache-logs-with-fluentd-and-Minio-server.md

```diff
@@ -43,12 +48,12 @@ We are replacing `/etc/td-agent/td-agent.conf` with:
    buffer_chunk_limit 256m
 </match>
 ```
-After this please restart your `fluentd` server.  
+Step 3 : Restart `fluentd` server.  
 ```
```

Comment:
Should be Step3: 
No space between 3 and : 

Similarly correct other steps aswell.